realis.go

```diff
@@ -688,7 +688,7 @@ func (r *realisClient) CreateService(
 		return resp, resp.GetResult_().GetStartJobUpdateResult_(), nil
 	}
 
-	return nil, nil, errors.New("results object is nil")
+	return nil, nil, errors.New("job is unchanged by proposed update")
```

Comment:
`Job is unchanged by proposed update` should be part of the result or response as following
``` [TRACE] Aurora Thrift Call ended resp: Response({ResponseCode:OK Result_:<nil> ServerInfo:ServerInfo({ClusterName:gpf-dev StatsUrlPrefix:}) Details:[ResponseDetail({Message:Job is unchanged by proposed update.})]}) clientErr: <nil>```

we should leave the error `results object is nil` as it is.

Reply:
Thanks for the clarification.